CoverflowAltTab@dmo60.de/locale/fr/LC_MESSAGES/coverflow.po

```diff
@@ -0,0 +1,88 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: CoverflowAltTab Gnome-Shell extension\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2016-01-05 00:06+0300\n"
+"PO-Revision-Date: 2016-01-05 00:06+0300\n"
+"Last-Translator: DAEM Q.\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.5.4\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+
+#: prefs.js:28 prefs.js:48
+msgid "Coverflow"
+msgstr ""
```

Comment:
Translation for Coverflow and Timeline is missing. My knowledge with po files is limited; does that mean that it will use the default ("Coverflow" and "Timeline") or is this an error and combobox is filled with the empty string?